cypress/e2e/specs/ethereum/connect-wallet.e2e-spec.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { tag } from '../../pages/swap-page.po.cy'
+import { homePage, network, wallet } from '../../selectors/selectors.cy'
+
+const chainList = [
+  'BNB',
+  'Polygon',
+  'Avalanche',
+  'Arbitrum',
+  'Optimism',
+  'BitTorrent',
+  'Oasis',
+  'Fantom',
+  'Cronos',
+  'Velas',
+  'Aurora',
+]
+
+describe('Metamask Extension tests', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.clickButton(homePage.skipTutorial)
+    cy.get(wallet.btnConnectWallet).should('be.visible').click()
+    cy.connectWallet()
+  })
+
+  it('Redirects to swap page when a user has already connected a wallet', { tags: tag.regression }, () => {
+    cy.acceptMetamaskAccess()
+    cy.get(wallet.statusConnected, { timeout: 10000 }).should('be.visible')
+    cy.url().should('include', '/swap')
+  })
+
+  it.skip('Should approve permission to switch network', { tags: tag.regression }, () => {
```

Comment:
```suggestion
  it('Should approve permission to switch network', { tags: tag.regression }, () => {
```

Reply:
Due to this [issue](https://github.com/Synthetixio/synpress/issues/761), I will check it later.